cmd/compute-domain-controller/main.go

```diff
@@ -47,6 +47,7 @@ const (
 )
 
 type Flags struct {
+	chartName        string
```

Comment:
I think this one makes the most sense right after imageName

Reply:
moved